numba/parfors/array_analysis.py

```diff
@@ -3148,7 +3185,19 @@ def _make_assert_equiv(self, scope, loc, equiv_set, _args, names=None):
             ir.Assign(value=value, target=var, loc=loc),
         ]
 
-    def _gen_shape_call(self, equiv_set, var, ndims, shape, post):
+    def _get_typ_part(self, typ, i):
+        if isinstance(typ, types.containers.UniTuple):
+            return typ.dtype
+        elif isinstance(typ, types.ArrayCompatible):
+            return types.intp
+        elif isinstance(typ, types.Number):
+            return typ
+        else:
+            if config.DEBUG_ARRAY_OPT >= 1:
+                print(1, "Unhandled case in _get_typ_part:", typ, i)
```

Comment:
```suggestion
                print("Unhandled case in _get_typ_part:", typ, i)
```
Was the 1 intentional?